pubspec.yaml

```diff
@@ -24,8 +22,15 @@ dependencies:
   yaml: ^2.1.16
 
 dev_dependencies:
-  dependency_validator: ^1.3.0
+  dependency_validator: ^2.0.1
   matcher: ^0.12.5
+  meta: ">=1.2.2 <1.7.0" # Workaround to avoid https://github.com/dart-lang/sdk/issues/46142
   test: ^1.15.7
   test_descriptor: ^1.2.0
   test_process: ^1.0.4
+
+dependency_validator:
+ exclude:
+  - test/tools/fixtures/sort_imports/imports.dart
+ ignore:
+  - meta
```

Comment:
Looking for guidance here. During the `format_and_publish` check:

```
Package validation found the following potential issue:
* "dependency_validator" is not a key recognized by pub - did you mean "dependency_overrides"?
```

Looks like we didn't add support for `dart_dependency_validator.yaml` until `dependency_validator` 3.0.0

Reply:
Yeah I think for this backpatch we should just disable the pub publish CI check.